sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2440,6 +2546,24 @@ public partial class ScheduledQueryRulePatch
         public System.TimeSpan? WindowSize { get { throw null; } set { } }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ScheduledQueryRulePublicNetworkAccess : System.IEquatable<Azure.ResourceManager.Monitor.Models.ScheduledQueryRulePublicNetworkAccess>
```

Comment:
Since your APIs are irrelevant to ScheduledQueryRule, this might introduce confusions.
Could we rename this to just `MonitorPublicNetworkAccess`?

Reply:
MonitorPublicNetworkAccess has also been mapped by `KnownPublicNetworkAccessOptions` already, but we can use MonitorWorkspacePublicNetworkAccess instead

Reply:
Appears to have been added here:
https://github.com/Azure/azure-sdk-for-net/commit/2868866226b79e9badbf74ab50dc5732b865b770#diff-a9beafcefa27a5fb6e197a8f2aed62cd019dbcd92eae37cb6e43b57144e79d79

Reply:
OK - this case looks like we are defining the same enum in different swaggers and happens to have the same name "PublicNetworkAccess"
Last time when we introduce APIs for scheduledQueryRules, we change it to this name `ScheduledQueryRulePublicNetworkAccess` since it is only used there.

Considering on the situation, let us leave it as is.

Reply:
Well I am kind of curious why this type is new here - since it is new, maybe we could change it

Reply:
Updated to MonitorWorkspacePublicNetworkAccess